soul-admin/src/main/java/org/dromara/soul/admin/listener/websocket/WebsocketCollector.java

```diff
@@ -68,8 +69,12 @@ public void onOpen(final Session session) {
     @OnMessage
     public void onMessage(final String message, final Session session) {
         if (message.equals(DataEventTypeEnum.MYSELF.name())) {
-            WebsocketCollector.session = session;
-            SpringBeanUtils.getInstance().getBean(SyncDataService.class).syncAll(DataEventTypeEnum.MYSELF);
+            try {
+                ThreadLocalUtil.put(SESSION_KEY, session);
+                SpringBeanUtils.getInstance().getBean(SyncDataService.class).syncAll(DataEventTypeEnum.MYSELF);
+            } finally {
```

Comment:
why remove?  it mean?

Reply:
After data synchronization, there is no need to keep the session,avoid OOM.